chain/client-primitives/src/types.rs

```diff
@@ -300,6 +300,17 @@ impl From<SyncStatus> for SyncStatusView {
     }
 }
 
+/// Implement Display through Debug for all variants but StateSync.
+/// For StateSync write only hash.
+impl std::fmt::Display for SyncStatus {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        match self {
+            SyncStatus::StateSync(hash, _) => write!(f, "StateSync {{ hash: {:?} }}", hash),
```

Comment:
Would it be possible to format `StateSync` with the use of this function (to be made public)?
https://github.com/near/nearcore/blob/a5ede1d1cd54906ac91cbb614028b5323ddee0c0/chain/client/src/client.rs#L2287

Reply:
Maybe the better way if to define `ShardSyncDownload::debug` using `format_shard_sync_phase()`